blocks/form/gdpr-confirmation.js

```diff
@@ -0,0 +1,80 @@
+import { readBlockConfig } from '../../scripts/lib-franklin.js';
+
+function hasQueryParams() {
+  return !!window.location.search;
+}
+
+// Send a query parameters to the GDPR API Accept
+async function acceptResult(accept, urlProd, sucessRedirect, block) {
+  // get url parameters
+  const urlParams = new URLSearchParams(window.location.search);
+  const paramData = urlParams.get('data');
+  const paramInit = urlParams.get('init');
+  const url = urlProd + accept;
+  const options = {
+    method: 'PUT',
+    headers: { 'Content-type': 'application/json; charset=UTF-8' },
+    body: JSON.stringify({
+      data: paramData,
+      init: paramInit,
+      confirm: 'true',
+    }),
+  };
+  try {
+    const response = await fetch(url, options);
+    const resData = await response.text();
+    if (response.ok) {
+      window.open(sucessRedirect);
+    } else {
+      const { message } = JSON.parse(resData);
+      throw message;
+    }
+  } catch (error) {
+    const divError = document.createElement('div');
+    divError.className = 'div-error';
```

Comment:
let's call this class simply `error` and change the css selectors to `.gdpr-confirmation .error { ... }` and `.gdpr-confirmation .close-error { ... }`